src/test/java/org/ays/auth/tests/UsersListTest.java

```diff
@@ -219,70 +220,64 @@ public void listUserByEmailAddress(){
         UserListPayload userListPayload = new UserListPayload();
         userListPayload.setPageable(AysPageable.generate(1, 10));
 
-        UserListPayload.Filter filter= new UserListPayload.Filter();
+        UserListPayload.Filter filter = new UserListPayload.Filter();
         filter.setEmailAddress(emailAddress);
 
-        List<String> statuses = new ArrayList<>();
-        statuses.add("ACTIVE");
-        statuses.add("PASSIVE");
-        filter.setStatuses(statuses);
+        filter.setStatuses(List.of(UserStatus.ACTIVE.name(), UserStatus.PASSIVE.name()));
 
         userListPayload.setFilter(filter);
 
-        Response response=UserEndpoints.findAll(userListPayload,accessToken);
+        Response response = UserEndpoints.findAll(userListPayload, accessToken);
         response.then()
                 .spec(AysResponseSpecs.expectSuccessResponseSpec())
-                .body("response.content[0].emailAddress",equalTo(emailAddress))
-                .body("response.content[0].id",notNullValue());
+                .body("response.content[0].emailAddress", equalTo(emailAddress))
+                .body("response.content[0].id", notNullValue());
 
     }
 
     @Test(groups = {"Regression"})
-    public void listAllUsersByEmailAddress(){
+    public void listAllUsersByEmailAddress() {
 
         LoginPayload loginPayload = LoginPayload.generateAsTestVolunteerFoundationAdmin();
         String accessToken = this.loginAndGetAccessToken(loginPayload);
 
         UserListPayload userListPayload = new UserListPayload();
         userListPayload.setPageable(AysPageable.generate(1, 10));
 
-        UserListPayload.Filter filter= new UserListPayload.Filter();
+        UserListPayload.Filter filter = new UserListPayload.Filter();
         filter.setEmailAddress("@gmail.com");
 
-        List<String> statuses = new ArrayList<>();
-        statuses.add("ACTIVE");
-        statuses.add("PASSIVE");
-        filter.setStatuses(statuses);
+        filter.setStatuses(List.of(UserStatus.ACTIVE.name(), UserStatus.PASSIVE.name()));
```

Comment:
Method değişince buradaki List.of kısmınında kalkması gerek

Reply:
@cagla88 Bu kullanımda ne gibi bir problem vardı?